### PR TITLE
JupyterLab extension

### PR DESCRIPTION
**Pull request by liuzhe-lz**
How to use:
1. `python setup.py develop`
2. `nnictl jupyter-extension install`
3. `jupyter lab --ip=0.0.0.0`
4. Create an experiment with nnictl or Python API
5. Click NNI button in JupyterLab launcher

### CODE REVIEW

File: nni/tools/jupyter_extension/__init__.py [hunk 1]
```diff
@@ -0,0 +1,4 @@
+from . import proxy
+
+load_jupyter_server_extension = proxy.setup
+_load_jupyter_server_extension = proxy.setup
```
Comment (J-shang) [inline nni/tools/jupyter_extension/__init__.py:4]: why `load_jupyter_server_extension` and `_load_jupyter_server_extension`, and where we use these?

Comment (liuzhe-lz) [reply nni/tools/jupyter_extension/__init__.py:4]: These are meta-info of jupyter extension, for different jupyter versions.

Comment (J-shang) [reply nni/tools/jupyter_extension/__init__.py:4]: got it.

**Comment by J-shang** [comment]
> How to use:
> 
> 1. `python setup.py develop`
> 2. `nnictl jupyter-extension install`
> 3. `jupyter lab --ip=0.0.0.0`
> 4. Create an experiment with nnictl or Python API
> 5. Click NNI button in JupyterLab launcher

Should we put `how to use` in doc or somewhere? Is it an internal feature?

**Comment by liuzhe-lz** [comment]
> > How to use:
> > 
> > 1. `python setup.py develop`
> > 2. `nnictl jupyter-extension install`
> > 3. `jupyter lab --ip=0.0.0.0`
> > 4. Create an experiment with nnictl or Python API
> > 5. Click NNI button in JupyterLab launcher
> 
> Should we put `how to use` in doc or somewhere? Is it an internal feature?

For now I don't want ordinal users to try it.

**Review by SparkSnail — APPROVED** [review]

**Review by J-shang — APPROVED** [review]

**Pull request merged**